.taskcluster.yml

```diff
@@ -0,0 +1,138 @@
+version: 1
+policy:
+  pullRequests: public
+tasks:
+  $let:
+    project_name: 'balrog'
```

Comment:
It looks like this is the only line that's different between treescript and balrogscript; that's a great sign.

Reply:
yes, that was the idea. we still need to figure out how to combine all of scriptworkers in one taskcluster.yml file. but that is a pain we can defer :)